docs/team/index.md

```diff
@@ -0,0 +1,73 @@
+title: Sematext Team and Access Sharing
+description: There are two levels of access rights. Account and App Access. Inviting team members to your account means they get access to all your Apps, Dashboards, notification hooks, alert rules, and everything else! You can invite team members to individual Apps and access Apps you’ve been invited to. 
+
+
+There are two levels of access rights: 
+
+- **Account Access** - invite others to your **whole account**
+- **App Access** - invite others **only to a particular App**
+
+And a third option called:
+
+- **App Transfer** - transferring ownership status of an App to another user
+
+You can assign three different Roles for users you give access to your account 
+or particular Apps:
+
+- **User**
+- **Admin**
+- **Billing Admin**
+
+And a fourth option: 
+
+- **Owner** - assigned if you transfer the ownership of an App to another user.
+
+
+## Account vs. App Access
+Account Access and App Access are not exclusive. You can use both of
+these two access types at the same time. You could share your Account with
+some users, and share specific Apps with other users.
+
+## Account Access
+By inviting a user to your Account, you invite them to your **whole account**, 
+so they get access to **all** your Apps, Dashboards, notification
+hooks, alert rules, and integrations.
+
+They can also create new Apps under your account and invite other users. 
+Depending on the role you assign to invitees, they may be able to administer 
+your Apps, Dashboards, users and even billing info (change app plan, assign 
+credit card, etc). 
+
+Account Sharing is very convenient because as soon as a new
+Sematext App is added to your account or new Dashboard is created, all
+users added to your account get access to it. Of course, the level of
+access depends on the role you initially assigned to each person.
```

Comment:
drop "initially", because it sounds like you can assign the role only initially and cannot change it.